zk/stages/stage_l1syncer.go

```diff
@@ -230,6 +230,10 @@ func parseLogType(l1RollupId uint64, log *ethTypes.Log) (l1BatchInfo types.L1Bat
 		batchLogType = logVerify
 		batchNum = new(big.Int).SetBytes(log.Topics[1].Bytes()).Uint64()
 		stateRoot = common.BytesToHash(log.Data[:32])
+	case contracts.VerificationValidiumTopicEtrog:
+		batchLogType = logVerify
```

Comment:
Shouldn’t we take rollupID into account here as well?

Reply:
Good catch! Will add the check.